### ISSUE TITLE
Tables for labels and categories?

### ISSUE THREAD

**Issue by vrom911** [kowainik/issue-wanted#139]

**Comment by rashadg1030** [comment]
@vrom911 Yes good question. Originally we did have a table for labels and categories, but I think we decided we didn't need it and it complicated the database schema, so we removed it...I forgot the exact reason why. If you look at the code from earlier in the project you can see it. Should we add this back into the code?

**Comment by rashadg1030** [comment]
@chshersh @vrom911 Bump

**Comment by rashadg1030** [comment]
@chshersh Okay got it makes sense. Thank you for explaining. I'm ready to tackle this issue, but should we figure out how to optimize the syncing algorithm first?

**Comment by rashadg1030** [comment]
@chshersh @vrom911 So for GitHub labels are we just selecting a few labels that we think are useful or do we plan on fetching all the labels on GitHub? I'm assuming the former. In that case, which labels should we select?

**Comment by chshersh** [comment]
@rashadg1030 Sure, let's finish with the sync first and think about proper caching. Not sure how it should be done, but probably one more step after syncing.
